frontend/public/locales/en/translation.json

```diff
@@ -27,6 +28,8 @@
   "{{remaining}} more": "{{remaining}} more",
   "{{remoteCount}} Remote": "{{remoteCount}} Remote",
   "{{remoteCount}} Remote, 1 Local": "{{remoteCount}} Remote, 1 Local",
+  "{{testInventory}} is not an existing ansible inventory": "{{testInventory}} is not an existing Ansible inventory",
+  "{{testSecret}} is not an existing ansible credential": "{{testSecret}} is not an existing Ansible credential",
```

Comment:
Might as well make the key match the value since it is a new message. The difference in case might be confusing.